go/engine/bootstrap.go

```diff
@@ -86,38 +85,6 @@ func (e *Bootstrap) Run(m libkb.MetaContext) error {
 	e.status.Username = e.G().ActiveDevice.Username(m).String()
 	m.Debug("Bootstrap status: uid=%s, username=%s, deviceID=%s, deviceName=%s", e.status.Uid, e.status.Username, e.status.DeviceID, e.status.DeviceName)
 
-	// get user summaries
-	ts := libkb.NewTracker2Syncer(e.G(), e.status.Uid, true)
-	if e.G().ConnectivityMonitor.IsConnected(context.Background()) == libkb.ConnectivityMonitorYes {
-		m.Debug("connected, loading self user upak for cache")
-		arg := libkb.NewLoadUserArgWithMetaContext(m).WithUID(e.status.Uid)
-		if _, _, err := e.G().GetUPAKLoader().Load(arg); err != nil {
-			m.Debug("Bootstrap: error loading upak user for cache priming: %s", err)
-		}
```

Comment:
this isn't needed anymore?

Reply:
No, I think it is a waste. The purpose was to prime the UPAK cache with yourself, but you end up just slowing down startup to do it.